integrations/postgresql/sensu-integration.yaml

```diff
@@ -0,0 +1,115 @@
+---
+api_version: catalog/v1
+type: Integration
+metadata:
+  namespace: postgresql
+  name: postgresql-health-check
+spec:
+  class: enterprise # community, supported, enterprise, or partner
```

Comment:
Is this enterprise? Or is this supported?

Reply:
Wouldn't this apply only to enterprise @jspaleta ?

Reply:
"enterprise" class is for integrations that require a valid license (mostly handler integrations)

this should be "supported" class